release-notes/Cloud/DCP_change_log.md

```diff
@@ -9,6 +9,30 @@ The dataminer.services platform gets updated continuously. This change log can h
 > [!NOTE]
 > Many features on dataminer.services are dependent on DxMs. You can find the change logs for these under [DxM release notes](xref:DxM_RNs_index).
 
+#### 16 October 2024 - Enhancement - Web apps - Colors web apps adjusted [ID 41100]
+
+​The dataminer.services web apps have been updated to make use of new colors, which will provide better contrast and make the UI easier to read.
+
+#### 16 October 2024 - Enhancement - Account linking & Catalog - 'Logout' changed to 'Sign out' [ID 41024]
+
+In the account linking and Catalog UI, the term "Logout" has been replaced with "Sign out".
+
+#### 16 October 2024 - Fix - Catalog - Deploy trial licensed issues [ID 41024]
+
+Deploy (trial) button in the header on the catalog details page no longer mentions not licensed issue.
+
+#### 16 October 2024 - Enhancement - Catalog - Description styling [ID 40965]
+
+The styling of alerts in ​Catalog descriptions has been adjusted so it matches the styling of [docs.dataminer.services](https://docs.dataminer.services). The currently supported alert types are caution, important, note, tip, and warning.
+
+#### 16 October 2024 - Enhancement - Catalog - Design improvements [ID 40965]
+
+The card layout in the Catalog has been redesigned to be more concise. The detail page header section, search results, and home page sections have been updated to match the new card design.
+
+#### 16 October 2024 - Fix - Sharing - Missing emails [ID 41110]
+
+On the sharing edit page. Email addresses of people who have access to the share are now correctly filled in.
```

Comment:
Same problem as for RN 41024: Please rephrase this description so it matches the phrasing of other fixes. The focus should be on describing the issue.